benchmarks/mruby-2018-05-23/Dockerfile

```diff
@@ -17,7 +17,10 @@
 FROM gcr.io/oss-fuzz-base/base-builder@sha256:fb1a9a49752c9e504687448d1f1a048ec1e062e2e40f7e8a23e86b63ff3dad7c
```

Comment:
Realize that if this is pinned to an Ubuntu 16 image then you haven't upgraded the image this is being built in.

Reply:
Sorry, I am not sure if I understood: Why is it pinned to 16?
The hash is `Ubuntu 20.04.5 LTS`.

Reply:
Nevermind youre right.